doc/api/deprecations.md

```diff
@@ -660,6 +660,15 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: path._makeLong()
+
+Type: Runtime
```

Comment:
@jasnell I think I have two issues with that approach:
- There should to be a time frame for migration, in which using either name just works, and a run-time deprecation *is* functional breakage
- This PR is committing to supporting this API anyway (which is a good thing in my eyes!). Imho, we should focus runtime-deprecation of APIs, internal or not, on code that can actually be expected to be broken/removed in the future.